next/components/forms/simple-components/AriaMenuComponent.tsx

```diff
@@ -0,0 +1,32 @@
+import type { ItemProps, MenuProps, MenuTriggerProps } from 'react-aria-components'
+import { Item } from 'react-stately'
+
+interface MyMenuButtonProps<T> extends MenuProps<T>, Omit<MenuTriggerProps, 'children'> {
+  label?: string
+}
+// const MyItem = (props: ItemProps) => {
+//   return (
+//     <Item
+//       {...props}
+//       className={({ isFocused, isSelected }) => `my-item ${isFocused ? 'focused' : ''}`}
+//     />
+//   )
+// }
+
+const AriaMenuComponent = <T extends object>({
```

Comment:
Do you use this component? If not, please remove and uninstall `react-aria-components` lib, if not used.